scripts/bee.sh

```diff
@@ -149,6 +157,12 @@ do
         BEE_PASSWORD="${1#*=}"
         shift 1
         ;;
+        --restrict=*)
+        RESTRICTED="true"
+        RESTRICTED_PASSWORD="${1#*=}"
+        RESTRICTED_PASSWORD_HASH=$(htpasswd -bnBC 10 "" $RESTRICTED_PASSWORD | tr -d ':\n')
```

Comment:
it requires to install `apache2-utils`.

can we generate the password hash with `openssl passwd` or some other tool that is available by default?

Reply:
Yeah, you are right, but I have not discovered any other way. I don't think `openssl` supports `bcrypt`. I have asked if Bee team could implement simple CLI command that would provide this hashing functionality, but until then I would go with this...

Reply:
Okay, then please add this as a dependency to the project until then

Reply:
I assume by "add this as a dependency" you mean to document that in README right?